crates/pink/pink-extension-runtime/src/mock_ext.rs

```diff
@@ -155,6 +156,10 @@ impl ext::PinkExtBackend for MockExtension {
     fn js_eval(&self, codes: Vec<JsCode>, args: Vec<String>) -> Result<JsValue, Self::Error> {
         super::DefaultPinkExtension::new(self).js_eval(codes, args)
     }
+
+    fn worker_attestation(&self) -> Result<AttestationResult, Self::Error> {
+        Ok(Ok(None).into())
```

Comment:
shall we provide a mocked (maybe invalid) report instead of None?

Reply:
Sure, Added.